web/src/recent_view_ui.ts

```diff
@@ -52,11 +52,7 @@ import * as user_status from "./user_status";
 import * as user_topics from "./user_topics";
 import * as views_util from "./views_util";
 
-type Row = {
-    last_msg_id: number;
-    participated: boolean;
-    type: "private" | "stream";
-};
+type Row = ConversationData;
```

Comment:
Do we need to define this type if they are the same?

Reply:
Could be an additional commit to clean that up if that's convenient.

Reply:
I wasn't sure here -- they were separate types before despite being the same. Was there a reason for that? I thought maybe there was a semantic reason for it or something. Would you prefer they both be called `ConversationData`?